rpc/http.go

```diff
@@ -208,6 +213,28 @@ type httpServerConn struct {
 	r *http.Request
 }
 
+func newDecodeCompression(decoding string, rc io.ReadCloser) (io.ReadCloser, error) {
+	tps := strings.Split(strings.TrimSpace(strings.ToLower(decoding)), ",")
+	var res io.ReadCloser
+	switch tps[0] {
+	case "gzip":
+		gz, err := gzip.NewReader(rc)
+		if err != nil {
+			return nil, err
+		}
+		res = gz
+	case "zlib", "deflate":
```

Comment:
Based on [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding), `zlib` does not seem to be a standard encoding. (`"deflate"` means zlib, not `"zlib"`)

Reply:
fixed